scopes/dependencies/pnpm/pnpm.package-manager.ts

```diff
@@ -114,6 +115,7 @@ export class PnpmPackageManager implements PackageManager {
       proxyConfig,
       networkConfig,
       {
+        nodeLinker: config.nodeLinker,
```

Comment:
I would get this value first from the pnpm config in the workspace.jsonc
to do it, all you need to do is get another argument in the pnpm.main.runtime provider which called config (right after the deps array) - see the dep-resolver.main.runtime.
This config will auto include everything that configured under the pnpm aspect in the ws.jsonc.
If it exists take it from there. only if not take it from the npmrc config.